R/convert_stat_chisq.R

```diff
@@ -1,10 +1,9 @@
 #' Conversion Chi-Squared to Phi or Cramer's V
 #'
 #' Convert between Chi square (\eqn{\chi^2}), phi (\eqn{\phi}), Cramer's V,
-#' Cohen's *w*, Fei (*פ*) and Pearson's *C* for contingency tables or
```

Comment:
What symbol is that?

Is that Hebrew letter _pe_?
https://en.wiktionary.org/wiki/%D7%A4

Reply:
Yes - the letter can be pronounced as P or as F; when pronounced as F, the letter is called Fe; in Hebrew that last vowel usually sounds more like the "ei" diphthong, and so - Fei.

Thanks' for coming to my Hebrew lesson (: